docs/root/start/sandboxes/dynamic-configuration-filesystem.rst

```diff
@@ -0,0 +1,120 @@
+.. _install_sandboxes_dynamic_config_fs:
+
+Dynamic configuration (filesystem)
+==================================
+
+This example walks through configuring Envoy using filesystem-based dynamic configuration.
+
+It demonstrates how configuration provided to Envoy dynamically can be updated without
+restarting the server.
+
+.. include:: _include/docker-env-setup.rst
+
+Change directory to ``examples/dynamic-config-fs`` in the Envoy repository.
+
+Step 3: Start the proxy container
+*********************************
+
+.. note::
+
+   If you are running on a system with strict ``umask`` you will need to ``chmod`` the dynamic config
+   files which are mounted into the container:
+
+   .. code-block:: console
+
+      $ umask
+      027
+      $ pwd
+      envoy/examples/dynamic-config-fs
+      $ chmod go+r configs/*
+      $ chmod go+x configs
+
+Build and start the containers
+
+This should also start two backend ``HTTP`` echo servers, ``service1`` and ``service2``.
+
+.. code-block:: console
+
+    $ pwd
+    envoy/examples/dynamic-config-fs
+    $ docker-compose build --pull
+    $ docker-compose up -d
+    $ docker-compose ps
+
+           Name                            Command                State                     Ports
+    ------------------------------------------------------------------------------------------------------------------------
+    dynamic-config-fs_proxy_1      /docker-entrypoint.sh /usr ... Up      0.0.0.0:10000->10000/tcp, 0.0.0.0:19000->19000/tcp
+    dynamic-config-fs_service1_1   /bin/echo-server               Up      8080/tcp
+    dynamic-config-fs_service2_1   /bin/echo-server               Up      8080/tcp
+
+Step 4: Check web response
+**************************
+
+You should be able to make a request to port ``10000``, which will be served by ``service1``.
+
+.. code-block:: console
+
+   $ curl -s http://localhost:10000
+   Request served by service1
+
+   HTTP/2.0 GET /
```

Comment:
im confused as to why this is getting `HTTP/2.0` but the control plane sandbox, didnt

Reply:
Hmm no clue.